redisinsight/ui/src/pages/browser/components/bulk-actions/BulkActionsInfo/BulkActionsInfo.styles.tsx

```diff
@@ -0,0 +1,117 @@
+import styled from 'styled-components'
+import React from 'react'
+import { ColorText, Text } from 'uiSrc/components/base/text'
+import { Theme } from 'uiSrc/components/base/theme/types'
+import { getApproximatePercentage, Maybe } from 'uiSrc/utils'
+import { isUndefined } from 'lodash'
+import { isProcessedBulkAction } from 'uiSrc/pages/browser/components/bulk-actions/utils'
+import { BulkActionsStatus } from 'uiSrc/constants'
+import { Props } from './BulkActionsInfo'
+
+export const BulkActionsInfoFilter = styled.div<{
+  className?: string
+  children?: React.ReactNode
+}>`
+  display: inline-flex;
+  gap: ${({ theme }) => theme.core.space.space050};
+  align-items: center;
+  font-size: ${({ theme }) => theme.core.font.fontSize.s10};
+`
+
+export const BulkActionsTitle = styled(Text).attrs({
+  size: 'XS',
+})<React.ComponentProps<typeof Text> & { $full?: boolean }>`
+  color: ${({ theme, color }) =>
+    !color && theme.semantic.color.text.informative400};
+  ${({ $full }) => $full && 'width: 100%'}
+`
+
+export const BulkActionsInfoSearch = styled(ColorText).attrs({
+  size: 'XS',
+})`
+  word-break: break-all;
+`
+
+export const BulkActionsProgress = styled(BulkActionsTitle)<{
+  $completed?: boolean
+}>`
+  position: absolute;
```

Comment:
nit: It just caught my eye, but do we actually need absolute positioning for this element?